server/github.go

```diff
@@ -158,22 +159,42 @@ func (s *Server) getFilenamesInPullRequest(pr *model.PullRequest) ([]string, err
 	return filenames, nil
 }
 
-func (s *Server) checkUserPermission(user, repoOwner string) bool {
-	_, resp, err := s.GithubClient.Organizations.GetOrgMembership(context.Background(), user, repoOwner)
-	if resp.StatusCode == 404 {
-		mlog.Info("User is not part of the ORG", mlog.String("User", user))
-		return false
+func (s *Server) getMembers(ctx context.Context) (orgMembers []string, err error) {
+	opts := &github.ListMembersOptions{
+		ListOptions: github.ListOptions{PerPage: 30},
```

Comment:
Nit pick: I'm not sure if it's worth to specify `PerPage`. The library usually "knows best" what to pick.
```suggestion
		ListOptions: github.ListOptions{},
```